djautotask/sync_rest.py

```diff
@@ -781,6 +804,203 @@ def _assign_field_data(self, instance, object_data):
+        'status': (models.ServiceCallStatus, 'status'),
+        'creatorResourceID': (models.Resource, 'creator_resource'),
+        'canceledByResourceID': (models.Resource, 'canceled_by_resource')
+    }
+
+    @property
+    def active_ids(self):
+        active_ids = models.Account.objects.filter(
+            active=True
+        ).values_list('id', flat=True).order_by(self.lookup_key)
+
+        return active_ids
+
+    def _assign_field_data(self, instance, object_data):
+        instance.id = object_data['id']
+        instance.description = object_data.get('description')
+        instance.duration = Decimal(str(object_data.get('duration')))
+        instance.complete = object_data.get('isComplete')
+        instance.create_date_time = object_data.get('createDateTime')
+        instance.start_date_time = object_data.get('startDateTime')
+        instance.end_date_time = object_data.get('endDateTime')
+        instance.canceled_date_time = object_data.get('canceledDateTime')
+        instance.last_modified_date_time = \
+            object_data.get('lastModifiedDateTime')
+
+        self._set_datetime_attribute(instance, 'create_date_time')
+        self._set_datetime_attribute(instance, 'start_date_time')
+        self._set_datetime_attribute(instance, 'end_date_time')
+        self._set_datetime_attribute(instance, 'canceled_date_time')
+        self._set_datetime_attribute(instance, 'last_modified_date_time')
+
+        self.set_relations(instance, object_data)
+
+        return instance
+
+    def create(self, **kwargs):
+        """
+        Make a request to Autotask to create an entity.
+        """
+        instance = self.model_class()
+        api_client = self.client_class()
+        created_id, _ = api_client.create(instance, **kwargs)
+
+        # get_single retrieves other generated/calculated fields info
+        # from AT-side
+        created_instance = self.get_single(created_id)
+
+        return self.update_or_create_instance(created_instance['item'])
+
+
+class ServiceCallTicketSynchronizer(BatchQueryMixin, Synchronizer):
+    client_class = api.ServiceCallTicketsAPIClient
+    model_class = models.ServiceCallTicketTracker
+    condition_field_name = 'ticketID'
+    last_updated_field = None
+    force_batch_query_size = 240
+
+    related_meta = {
+        'serviceCallID': (models.ServiceCall, 'service_call'),
+        'ticketID': (models.Ticket, 'ticket')
+    }
+
+    @property
+    def active_ids(self):
+        active_ids = models.Ticket.objects.all().\
+            values_list('id', flat=True).order_by(self.lookup_key)
+
+        return active_ids
+
+    def _assign_field_data(self, instance, object_data):
+        instance.id = object_data['id']
+        self.set_relations(instance, object_data)
+
+        return instance
+
+    def create(self, **kwargs):
+        kwargs.update({
+            'parent': kwargs.get('service_call')
+        })
+        return super().create(**kwargs)
```

Comment:
parent should be passed in as an argument like on the update method, we shouldn't need to update `kwargs`. Its part of the childapimixin so we always know a parent will exist.